2gp-demo/config/developer-scratch-def.json

```diff
@@ -1,16 +1,11 @@
 {
-  "orgName": "PLP Dev Perview Co",
+  "orgName": "PLP Dev Preview (2GP Developer)",
   "edition": "Developer",
   "features": ["PartnerLicensingPlatform"],
   "settings": {
     "lightningExperienceSettings": {
       "enableS1DesktopEnabled": true
     },
-    "securitySettings": {
```

Comment:
reason for this removal?

Reply:
The setting `enableSetPasswordInApi` was deprecated in API v. 51.0 and removed in 52.0